homeassistant/components/progettihwsw/__init__.py

```diff
@@ -44,3 +46,13 @@ def setup_input(api: ProgettiHWSWAPI, input_number: int) -> Input:
 def setup_switch(api: ProgettiHWSWAPI, switch_number: int, mode: str) -> Relay:
     """Initialize the output pin."""
     return api.get_relay(switch_number, mode)
+
+
+def setup_temperature(api: ProgettiHWSWAPI, input_number: int) -> Temperature:
+    """Initialize the output pin."""
+    return api.get_temp(input_number)
+
+
+def setup_analog(api: ProgettiHWSWAPI, input_number: int) -> AnalogInput:
+    """Initialize the output pin."""
+    return api.get_pot(input_number)
```

Comment:
These are only used in sensors? Why are they part of the main module in this case?

As they contain a single method call, is it really needed to abstract this into a new method?